src/settings/utils/template-resolver.js

```diff
@@ -2,7 +2,7 @@ export default function (str) {
   return (o) => {
     return str.replace(/{{([^{}]*)}}/g, (a, b) => {
       const r = o[b];
-      return typeof r === 'string' || typeof r === 'number' ? r : a;
+      return typeof r === 'string' || typeof r === 'number' ? r : '';
```

Comment:
Since you edited stuff here, could you please give more descriptive variable names to `o`, `r`, and maybe `a` and `b` as it is hard to reason about them.